profiles/monodroid/System.Net.Http.cs

```diff
@@ -19,6 +19,7 @@
 [assembly:System.Runtime.CompilerServices.InternalsVisibleToAttribute("System.Net.Http.WebRequest, PublicKey=002400000480000094000000060200000024000052534131000400000100010007d1fa57c4aed9f0a32e84aa0faefd0de9e8fd6aec8f87fb03766c834c99921eb23be79ad9d5dcc1dd9ad236132102900b723cf980957fc4e177108fc607774f29e8320e92ea05ece4e821c0a5efe8f1645c4c0c93c1ab99285d622caa652c1dfad63d745d6f2de5f17e5eaf0fc4963d261c8a12436518206dc093344d5ad293")]
 [assembly:System.Runtime.CompilerServices.RuntimeCompatibilityAttribute(WrapNonExceptionThrows=true)]
 [assembly:System.Runtime.InteropServices.ComVisibleAttribute(false)]
+[assembly:System.Security.Permissions.SecurityPermissionAttribute(System.Security.Permissions.SecurityAction.RequestMinimum, SkipVerification=true)]
```

Comment:
Does this come from CoreFX? Generally speaking it's obsolete and should be removed. I thought it was already done globally on the CoreFX side.

Reply:
It's certainly possible.  I should probably rerun the api snapshot tool to make sure we're actually using the latest version.